Datacard/makeDatacardGGTT_new.py

```diff
@@ -415,9 +415,16 @@ def main(args):
     td.writePdfIndex(f,df[~df.cat.isin(cr_cats)],opt,"_combined")
 
     f.write("\nsignal_scaler rateParam * %s* 0.001\nnuisance edit freeze signal_scaler"%(args.procTemplate))
+    HHRateParamWritten = False;
+    ResRateParamWritten = False;
     for proc in df[df.prune==0].proc.unique():
```

Comment:
Hi Manos, I have a minor question here. If we loop over the proc and only fill in the HH parameter and ResRateParameter once, why do we need a loop? 
I'm also curious about that, in the if "M125" statement, do we only put one of the 4 resonant background "ggh_M125", "tth_M125"....? Maybe I'm understanding incorrectly, but thanks in advance for your explanation

Reply:
We will do it only once, that is the point of these booleans. But, especially if we are pruning, we need to check if the relevant processes are there. If you have better ideas about how to do that without a loop, let me know.
As for the "M125", we use the same rate parameter to control all of the single Higgs processes. The rate parameter is coded in the datacards as `*M125*`, which includes all of the processes and not just a specific one.
I hope the above answers your questions.

Reply:
Thank you for explaining!